specification/botservice/resource-manager/Microsoft.BotService/preview/2021-05-01/botservice.json

```diff
@@ -0,0 +1,2976 @@
+            "schema": {
+              "$ref": "#/definitions/ConnectionSettingResponseList"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed. If the resource group does not exist, 404 (NotFound) will be returned.",
+            "schema": {
+              "$ref": "#/definitions/Error"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.BotService/hostSettings": {
+      "get": {
+        "tags": [
+          "BotHostSettings"
+        ],
+        "description": "Get per subscription settings needed to host bot in compute resource such as Azure App Service",
+        "operationId": "HostSettings_Get",
+        "x-ms-examples": {
+          "Get Bot Host Settings": {
+            "$ref": "./examples/GetHostSettings.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "If resource is retrieved successfully, the service should return 200 (OK).",
+            "schema": {
+              "$ref": "#/definitions/HostSettingsResponse"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed",
+            "schema": {
+              "$ref": "#/definitions/Error"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.BotService/operationresults/{operationResultId}": {
+      "get": {
+        "tags": [
+          "OperationResults"
+        ],
+        "description": "Get the operation result for a long running operation.",
+        "operationId": "OperationResults_Get",
+        "x-ms-examples": {
+          "Get operation result": {
+            "$ref": "./examples/OperationResultsGet.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/operationResultIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The body contains all of the properties of the operation result.",
+            "schema": {
+              "$ref": "#/definitions/OperationResultsDescription"
+            }
+          },
+          "202": {
+            "description": "Accepted - Get request accepted; the operation will complete asynchronously."
+          },
+          "404": {
```

Comment:
>404 [](http://example.com/codeflow?start=11&length=3)

It's common to let this default, your choice.

Reply:
removed to let it default